sprintly/sprintly-ui
# Doc additions revisions

#### What's this PR do?
Adds a lot of new documentation on code and process for using and contributing to Sprintly-ui.

#### Where should the reviewer start?
Start with the readme and contributing docs

#### How should this be manually tested?
Read. Could also pull down the branch and check out the new fuller-featured examples by running ```$ npm run build-css && npm run dev```

#### Any background context you want to provide?
nope

#### What ticket or issue # does this fix?
Fixes #9548

#### Screenshots (if appropriate)
New interactive menu example page:
![screen shot 2015-03-05 at 5 29 30 pm](https://cloud.githubusercontent.com/assets/1137259/6816812/3df89a12-d255-11e4-85fd-272734e70389.png)

#### What gif best describes this PR or how it makes you feel?
![giphy](https://cloud.githubusercontent.com/assets/1137259/6816849/ca8cf392-d255-11e4-950c-da4788e0a730.gif)

#### Definition of Done:
- [ ] Does this contain breaking changes?
- [ ] Did you bump the version number accordingly?
- [ ] Will this feature require a new piece of infrastructure be implemented?
- [ ] Did you update the documentation per these changes?
